hack/update-deps.sh

```diff
@@ -28,7 +28,8 @@ TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
 cd "${TESTINFRA_ROOT}"
 
 trap 'echo "FAILED" >&2' ERR
-bazel run //:dep -- ensure -v
+rm -rf vendor/github.com/golang/dep/internal/fs/testdata
+bazel run //:dep -- ensure -v "$@"
```

Comment:
I was going to ask if the comment for the script needed updating for this functionality, but then I noticed it's describing `prune-libraries.sh`, not `update-deps.sh`.

in any case, maybe update that comment? probably worth updating `docs/dep.md` too.

Reply:
Done